README.md

```diff
@@ -46,7 +46,7 @@ There are several ways to `Provide` an object:
 1. Provide a pointer to an existing object
 1. Provide a slice, map, or array
 
-### Provide a Constructor
+### Provide a constructor
```

Comment:
I think we agreed to do Title Case for H1s and Sentence case for H2s and beyond.